Library/Formula/fmilib.rb

```diff
@@ -0,0 +1,41 @@
+class Fmilib < Formula
+  desc "FMI Library enables integration of Functional Mock-up Units (FMUs)"
+  homepage "http://www.jmodelica.org/FMILibrary"
+  url "http://www.jmodelica.org/downloads/FMIL/FMILibrary-2.0.1-src.zip"
+  sha256 "2b82ad1bbb54399ef595755ca9e12e678e8d1ad0e227609d478d9934a298cc43"
+  head "https://svn.jmodelica.org/FMILibrary/trunk"
+
+  option "with-debug", "Build with debugging symbols"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = %W[
```

Comment:
The `std_cmake_args` includes a prefix argument, so this shouldn't be necessary.  You should be able to do something like
```ruby
args = std_cmake_args
args << "-DFMILIB_DEFAULT_BUILD_TYPE_RELEASE=false" if build.with? "debug"